modules/installing-rhv-example-install-config-yaml.adoc

```diff
@@ -136,3 +136,72 @@ platform:
 pullSecret: '{"auths": ...}'
 sshKey: ssh-ed25519 AAAA...
 ----
+
+.Example: Adding a non-enforcing affinity group
```

Comment:
These non-standard titles are unfortunate, but it looks like you're just matching what's already in use elsewhere in the module, so... :shrug: for now.

Reply:
1. I am working with what's already here. So I appreciate the shrug :).
2. Do you have a suggestion on how to improve this?

Reply:
I guess there's two separate weird-to-me things going on with these examples (and I would vote for just pondering on these in a potential follow-up PR, and just match what's here for now).

Title-wise, I'd probably just go back to how they were previously before getting changed here:

https://github.com/openshift/openshift-docs/pull/22718/files#diff-5bcc51e3b3600eb8442a18767b763050d05853377f8520921b5089dafc9d313cL28-R26

That is, I don't think "Example:" needs a colon there, and "This is a" doesn't seem necessary for a title. If it's just an example file, then this seems good enough:

```
.Example `whatever.yaml` file
```

Or in your case with this line, just drop "Example" completely:

```
.Adding a non-enforcing affinity group
```

Structure-wise, the other non-standard thing that caught my eye was having the `.Title` separate from the code block for a few of them, and then having a paragraph of body text before the block. It's like it's trying to use `.Title`s like a subheading in a way. In that case, maybe things should be re-structured to use `[discrete]` headings instead. And here I'll affix another requisite :shrug: emoji.

Reply:
Fixed per your suggestion.